src/content/en/tools/chrome-devtools/network/reference.md

```diff
@@ -288,6 +295,7 @@ Below is a complete list of supported properties.
 * `status-code`. Only show resources whose HTTP status code match the
   specified code. DevTools populates the autocomplete dropdown menu with all
```

Comment:
Let's refer to either "dropdown" (line 281) or "autocomplete dropdown menu" consistently (pick one)